### DISCUSSION TITLE
How do I hide Container icons in tabs?

### DISCUSSION THREAD

**Discussion by alessandromassone** [mbnuqw/sidebery#1885] [Q&A]
I'm trying to hide the "Container" indicator/icons in tabs — the element controlled by .identity-color-toolbar in Firefox's horizontal tabs. I don't need them to show since I have a custom Container set for each Sidebery panel — which makes all the indicators superfluous.

I haven't found any documentation on how to customize them with the built-in Styles editor. Is it possible?

<img width="214" alt="Screenshot 2024-11-25 at 20 46 42" src="https://github.com/user-attachments/assets/5244df01-c153-4046-bcd8-daa7e2ff92a9">

**Comment by chrostino** [comment]
I'm also looking for a solution for this

**Comment by yawns** [comment]
I hid mine by adding this to the style editor in Sidebery's settings:

```
/* hide container indicator */
.Tab .ctx { display: none !important; }
```
